### DISCUSSION TITLE
I/O: ethics, policies and technologies for programmatic and open access to archaeological online data sets

### DISCUSSION THREAD

**Discussion by ddomizia** [archeofoss/archeofoss2022#4] [Panel proposals]
Panel proposal
##  I/O: ethics, policies and technologies for programmatic and open access to archaeological online data sets

![panel-af-3](https://user-images.githubusercontent.com/39925492/158991065-0f170701-422f-463f-ade6-d0f98d96cedb.jpeg)

Proponents:
- **Julian Bogdani**
  -  Sapienza University of Rome, Rome
  - [julian.bogdani@uniroma1.it](mailto:julian.bogdani@uniroma1.it)
  - [@JulianBogdani](https://twitter.com/JulianBogdani)
- **Riccardo Montalbano**
  - Università Telematica Internazionale Uninettuno
  - [ricca.montalbano@gmail.com](mailto:ricca.montalbano@gmail.com)
  - [@Riccard52906185](https://twitter.com/Riccard52906185)

The spread of web-based databases and information systems specifically designed for the archaeological domain has greatly enhanced data-collection and analysis processes. This trend has positively impacted on the availability of online data sets, most of them living in the so-called deep Web, due to copyright, privacy and/or security issues. Open-access repositories released by universities and other Public Administrations are making their way towards toward the broader public, who is sometimes also a contributor of archaeological structured information on the Web.

This panel aims at focusing on ethical and technical issues related with the open online programmatic access to archaeological data sets. Endpoints and APIs based on SOAP, REST, GraphQL, or SPARQL protocols have radically changed in the last decade the way we are using the Web, fostering a richer and more creative use and reuse of public data. 

We would like to welcome case-studies of data publishers and system designers who want to share their policy and actual technological solutions for the programmatic consume of the data sets they maintain. On the other hand, programmatic data consumers, such as the designers of client software, web-sites or applications based on the availability of archaeological datasets, or single researchers that make use of scripts and software that analyse remote archaeological data are welcomed as well.

**Comment by benducke** [comment]
This text is very hard to understand, as it does not use common English terminology for the things it describes.

If I understand correctly, then this session is about machine readable data publication and standardised, open APIs. That is a suitable topic for the conference. The title should be changed accordingly, and the abstract text needs some rewriting to make it more intelligible (and avoid terms like "Dark Web", which have an entirely different connotation).

**Reply by jbogdani** [reply]
Dear @benducke, thank you for the precious suggestions. Both the title and the panel presentations have been unpdated and simplified to better focus on the main topic.
